_posts/en/newsletters/2020-01-15-newsletter.md

```diff
@@ -0,0 +1,139 @@
+    period if the proposal is not activated; and---if it's clear that
+    the community still wants the proposal activated---force activation
+    using a BIP8 flag-day set to two years in the future (with faster
+    activation possible using versionbits signaling).  Node software can
+    prepare for this maximum 42 month process by including, even in its
+    initial versions, a configuration option users can manually enable
+    to enforce the BIP8 flag day if necessary.  If the first 18 months of
+    the activation period passes without activation (but also without
+    any blocking problems being discovered), new releases can enable
+    this option by default for the remaining 24 months of the activation
+    period.
+
+    In the responses to the post, [Jorge Timón][timon sf] and [Luke
+    Dashjr][dashjr sf] both proposed that any BIP8-like mechanism use
+    mandatory versionbits signaling leading up to the flag day (similar
+    to how [BIP148][] proposed to activate segwit); Corallo
+    [notes][corallo reply timon] that this conflicts with the third and
+    fourth goals.  Jeremy Rubin provides a [quick analysis][rubin sf] of
+    his previous [spork proposal][spork vid] (see [Newsletter #32][spork
+    summary]) in the context of the five goals.  Anthony Towns provides
+    [lucid commentary][towns sf] on several aspects of Corallo's
+    proposal and Timón's response.
+
+    No clear conclusion was reached in the thread and we suspect
+    discussion about activation methods will continue.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17578][] updates the `getaddressinfo` RPC to return an
+  array of strings corresponding to the labels used by an address.
+  Previously, it returned an array of objects where each label contained
+  fields for its name and its purpose, where purpose corresponded to the
+  part of the wallet that generated the address.  Users who need the old
+  behavior can use the `-deprecatedrpc=labelspurpose` configuration
+  parameter until Bitcoin Core 0.21.
+
+- [Bitcoin Core #16373][] causes the `bumpfee` RPC used for
+  [Replace-by-Fee][topic rbf] fee bumping to return a [Partially Signed
+  Bitcoin Transaction][topic psbt] (PSBT) when the user attempts to fee
+  bump a transaction using a wallet with disabled private keys.  The
+  PSBT can then be copied to an external wallet (such as a hardware
+  device or cold wallet) for signing.
+
+- [Bitcoin Core #17621][] fixes a potential privacy leak for users of
+  the `avoid_reuse` wallet flag.  This flag prevents the wallet from
+  spending bitcoins received to an address the wallet has already used
+  for spending (see [Newsletter #52][]).  This enhances privacy by
```

Comment:
style nit: Starting two consecutive sentences with "This ..." seems slightly inelegant to me.

Reply:
fixed